apps/condo/domains/ticket/hooks/useTicketTableFilters.tsx

```diff
@@ -228,9 +228,11 @@ export function useTicketTableFilters (): Array<FiltersMeta<TicketWhereInput>>
                 keyword: 'division',
                 filters: [filterProperty],
                 queryToWhereProcessor: (queryDivisions) => {
-                    // We define single division in the browser query as "propertyId1, propertyId2" ->
-                    // in GQLWhere we need ["propertyId1", "propertyId2"]
-                    return queryDivisions?.map(queryDivision => queryDivision.split(',')).flat(1)
+                    if (queryDivisions) {
+                        return queryDivisions
+                            .map(option => JSON.parse(option).value)
```

Comment:
What is option type here? Looks like there is a potential runtime error. May be add check for option type?

Reply:
In `searchOrganizationDivision` returned option value type is arrays of string.
The behavior of the selector is such that if the option value is not a string or a number, then the entire option object is cast to a string using `JSON.Stringify`

Added a try/catch here

Reply:
Try catch isn't a good solution) I would try to define the types more explicitly)